package.json

```diff
@@ -27,19 +27,24 @@
     "checkPackage": "yarn package-check",
     "format": "prettier --write \"./**/*.{js,jsx,json,ts,tsx}\" && sort-package-json \"package.json\" \"./*/package.json\"",
     "start": "yarn test --watch",
-    "test": "echo 'tests disabled'"
+    "test": "NODE_OPTIONS=--experimental-vm-modules NODE_ENV=\"test\" jest --no-watchman"
```

Comment:
Getting some import issues when trying to run the tests:

![image](https://user-images.githubusercontent.com/11248103/162795536-17596f72-bbf0-49a6-8886-fdb4cfc584a8.png)

Reply:
Are you 100% sure you npm installed before running?

Reply:
oh, I ran `yarn`. I could try `npm install`.